spec/models/post_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe Post, type: :model do
+  # tests go here
+  before :each do
+    @user = User.new(name: 'Madix', post_counter: 0)
+  end
+
+  it 'Title should always be present' do
+    post = Post.new(author: @user, Title: 'Some Test', comments_counter: 0, likes_counter: 0)
+    expect(post).to be_valid
+  end
+
+  it 'Title should be a maximum of 250 chars' do
+    post = Post.new(author: @user, Title: 'A' * 251, comments_counter: 0, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
+  it 'comments counter should not be negative' do
+    Post.new(author: @user, Title: 'A' * 251, comments_counter: -1, likes_counter: 0)
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments counter should be integer' do
+    Post.new(author: @user, Title: 'A' * 251, comments_counter: 'string', likes_counter: 0)
+    expect(subject).to_not be_valid
+
+    Post.new(author: @user, Title: 'A' * 251, comments_counter: true, likes_counter: 0)
+    expect(subject).to_not be_valid
+
+    Post.new(author: @user, Title: 'A' * 251, comments_counter: 3.21, likes_counter: 0)
```

Comment:
### Due to insufficient files changed I will leave a comment here.

- [ ] It is a requirement that we have unit tests for the `User`, `Post`, `Comment`, and `Like` models. You only have tests for two files, the Post and User model. Kindly add tests for the Like and Comment models as well.
![image](https://github.com/BonkeGcobo/blogAppRor/assets/81932478/80e2cc6d-ba77-4bb7-986e-c40ae0c5574e)